.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,6 +13,13 @@ Thanks for telling us about a problem with the pipeline.
 Please delete this text and anything that's not relevant from the template below:
 -->
 
+## Check Documentation
+
+I have checked the following places for your error:
+
+- [ ] [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
+- [ ] [nf-core/qbic-pipelines-cellranger pipeline documentation](https://nf-co.re/nf-core/qbic-pipelines-cellranger/usage)
```

Comment:
Should prob look to the usage doc on github here

Reply:
Thanks, fixed!